dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -14,76 +14,73 @@
 #NLS_ENCODING=UNICODE
 #NLS_MESSAGEFORMAT_NONE
 #
-
 logging.properties=Logging
 logging.properties.desc=Controls the capture and output of log and trace messages.
 
 #
 # File size / max files
 #
 max.file.size=Maximum log file size
-max.file.size.desc=Maximum size of a log file, in megabytes, before being rolled over; a value of 0 means no limit.
+max.file.size.desc=The maximum size (in MB) that a log file can reach before it is rolled. The Liberty runtime does only size-based log rolling. To disable this attribute, set the value to 0. The maximum file size is approximate. By default, the value is 20. Note: maxFileSize does not apply to the console.logfile.
 
 max.files=Maximum log files
-max.files.desc=Maximum number of log files that will be kept, before the oldest file is removed; a value of 0 means no limit.
+max.files.desc=Maximum number of log files that will be kept, before the oldest file is removed; a value of 0 means no limit. If an enforced maximum file size exists, this setting is used to determine how many of each of the logs files are kept. This setting also applies to the number of exception logs that summarize exceptions that occurred on any particular day. So if this number is 10, you might have 10 message logs, 10 trace logs, and 10 exception summaries in the ffdc/directory. By default, the value is 2. Note: maxFiles does not apply to the console.log file.
```

Comment:
Change
"Maximum number of log files that will be kept, before the oldest file is removed; a value of 0 means no limit. If an enforced maximum file size exists, this setting is used to determine how many of each of the logs files are kept. This setting also applies to the number of exception logs that summarize exceptions that occurred on any particular day."
to
"Maximum number of log files that are kept before the oldest file is removed; a value of 0 means no limit. If an enforced maximum file size exists, this setting is used to determine how many of each of the log files are kept. This setting also applies to the number of exception logs that summarize exceptions that occurred on a particular day."